### ISSUE TITLE
Categorize DNS Blocklists

### ISSUE THREAD

**Issue by TXC** [AdguardTeam/AdGuardHome#1958]
Possibility to categorize the blocklists for Ads, Malware, Phishing and Adult content.
Which in turn could populate the cards on the dashboard.

**Comment by KizunaH** [comment]
Duplicate of #1662

**Comment by TXC** [comment]
> Duplicate of #1662
> AI detection: false positive ;)

Not really, this is more to trigger for example the "Blocked malware/phishing" card with lists that AdGuard-Team ain't running, for example the "NoCoin Filter List" could be categorized under "Blocked malware/phishing"

Your request was related, yes, but not duplicate :)

**Comment by KizunaH** [comment]
@TXC yeah, it was a mistake. that is why explained below instead remove comment. :)

**Comment by iceman73** [comment]
You could offer a tag to the user for the List, and then use those tags to filter into the catagories

**Comment by jvoisin** [comment]
Another important usecase for this would be to have big scary alert for [stalkerwares]( https://github.com/Te-k/stalkerware-indicators) :)
